Create release-notes-march-2024.md

stef-cobaltio opened the pull request:
Creating a new page for Cobalts release notes for the month of March 2024

## Preview This Change

To see how this change looks in production, scroll down to **Deploy Preview**. Select the link that looks like `https://deploy-preview-<num>--cobalt-docs.netlify.app/`

## Variables

Help us support a “Write once, publish everywhere” single source of truth. If you see a line that looks like:

`{{% asset-categories %}}`

You’ve found a [shortcode](https://gohugo.io/content-management/shortcodes/) that we include in multiple documents.

You’ll find the content of the shortcode in the following directory:

https://github.com/cobalthq/cobalt-product-public-docs/tree/main/layouts/shortcodes

That shortcode has the same base name as what you see in the PR, such as `asset-categories.html`.

## Checklist for PR Author

[ ] Did you check for broken links and alt text?

Be sure to check for broken links and Alt Text issues. We have a partially automated process,
as described in this section of our repository README:
[Test Links and Alt Attributes](https://github.com/cobalthq/cobalt-product-public-docs/blob/main/README.md#test-links-and-alt-attributes).

mtsuji-cobalt reviewed (commented):
• We should update the March 2024 link in the main Product Updates page `content/en/Product Updates/_index.md` with /product-updates/release-notes-march-2024

• I think we should put the section on New Home Page Launches last, as the image is quite long and ppl might not scroll down to the bottom to see the New Insights Charts section

stef-cobaltio commented:
Great catch Maria - thank you! 

- I went in and added the page link for March 2024 to the Product Updates landing page and 
- I updated the order or the release notes to put the home page announcement at the bottom.
